src/pages/index.tsx

```diff
@@ -50,22 +54,33 @@ const Home: NextPage = () => {
   };
 
   useEffect(() => {
+    const controller = new AbortController();
+    const signal = controller.signal;
+
     // Initial fetch
-    const fetchAllData = async () => {
-      await fetchData("./status.json", setStatusData);
-      await fetchData("./incidents.json", setIncidentData);
-      await fetchData("./planned_maintenance.json", setPlannedMaintenanceData);
+    const fetchAllData = async (signal: AbortSignal) => {
+      await fetchData("./status.json", signal, setStatusData);
```

Comment:
It's not related to this PR, but there's no reason to use an async function & await here, as it means the requests will run serially rather than in parallel.

Reply:
`fetchAllData can remove the async function in here as `fetch` function returns a `Promise` instead of raw data and needs to be resolved.